### DISCUSSION TITLE
Multiple MCP23017 support?

### DISCUSSION THREAD

**Discussion by dnsre** [arendst/Tasmota#12417] [Ideas]
Hi,
I've been looking for a possible solution for Tasmota to support multiple MCP23017s at the same time.
Would like to have one MCP23017 at address  0x20 as 8 inputs and 8 outputs and have the possibility to add another MCP23017 at address 0x21 with 8 additional inputs and 8 additional outputs.
I would need some clues to modify the code to meet these requirements.
Thanks

**Comment by ascillato2** [comment]
Not supported at this moment.

You can check with the people in the old issue where this was asked before.

https://github.com/arendst/Tasmota/issues/5636

**Reply by sfromis** [reply]
You could also consider using ESP32 with more pins.

**Reply by barbudor** [reply]
Or PCF8574 which we can have up to 8 of them
